samples/qa-scrollernative/src/views/MainView.module.less

```
@@ -0,0 +1,22 @@
+body {
+	margin: 0;
+	position: fixed;
+	width: 100%;
+	height: 100%;
+	overflow: hidden;
+}
+
+.mainView {
+	display: flex;
+	flex-direction: column;
+	height: 100%;
+}
+
+.content {
+	display: flex;
+	padding-top: 12px;
+	box-sizing: border-box;
+	border: solid glay 10px;
+	width: 100%;
+	height: 100%;
```

I think it would be better to modify the height to prevent samples cut off. 
```suggestion
	height: 80%;
```

---

Suggested change.